### ISSUE TITLE
pkg-config SoQt doesn't have a proper link flag

### ISSUE THREAD

**Issue by Vanuan** [coin3d/soqt#72]
It looks like the 1.6.0 release doesn't have a proper `*.pc` file generated:

```
prefix=/usr
exec_prefix=${prefix}
bindir=${exec_prefix}/bin
libdir=${exec_prefix}/lib
includedir=${prefix}/include
datarootdir=${prefix}/share
datadir=${datarootdir}
docdir=${datarootdir}/doc/soqt
infodir=${datarootdir}/info
mandir=${datarootdir}/man
htmldir=${docdir}/html

Name: SoQt
Description: A Qt Gui-toolkit binding for Coin
Version: 1.6.0
Requires: Coin, Qt5Core, Qt5Gui, Qt5OpenGL, Qt5Widgets
Conflicts:
Libs: -L${libdir}  
Cflags: -I${includedir}  

frameworkdir=
qt_version=5.5.3
```

The `Libs:` line is missing `-lSoQt`.

**Comment by Vanuan** [comment]
I see there's a fix:
https://github.com/coin3d/soqt/commit/655d927f9b1c54c7ecdacc39aa15773f99d2d20d

But unfortunately, there's no newer version released. Could you tag and release a new version? Or branch it and release 1.6.1 with a hotfix?

**Comment by VolkerEnderlein** [comment]
Done. just released v1.6.1 that should include the fix. Thanks for your ongoing contributions to Coin development.
